src/http/client.cc

```diff
@@ -312,6 +312,15 @@ auto client::with_connection(Fn&& fn) {
     });
 }
 
+struct transport_error : public std::exception {
+    unsigned requests_submitted;
+    std::system_error lower;
```

Comment:
and may i learn what is "lower" is for? lower than what?

Reply:
Yup. It's the original exception that triggered this one. "Nested" is good term, but it's not really using the nested exceptions from std:: so I named it somehow else

Reply:
Renamed to `cause`